.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/rflperry/opt/miniconda3/envs/mvlearn-dev/bin/python"
+}
```

Comment:
to remove as the DSStore file.

I assume you do `git add .` it's quite dangerous